src/components/ColourSelector.tsx

```diff
@@ -0,0 +1,52 @@
+import {
+    Drawer,
+    DrawerBody,
+    DrawerHeader,
+    DrawerOverlay,
+    DrawerContent,
+    DrawerCloseButton,
+    useDisclosure,
+    IconButton,
+} from "@chakra-ui/react";
+import { EditIcon } from "@chakra-ui/icons";
+import ColourButtons from "./Colours";
```

Comment:
Inconsistent name of file, file name should match React component name (ColourButtons.tsx)